projects/addon-charts/components/bar-chart/bar-chart.component.ts

```diff
@@ -94,6 +94,7 @@ export class TuiBarChartComponent {
     private getMax(values: ReadonlyArray<readonly number[]>, collapsed: boolean): number {
         return collapsed
             ? Math.max(
+                  // eslint-disable-next-line no-restricted-syntax
```

Comment:
What happens if you add default `0` here?

Reply:
<img width="1153" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/2c899833-86bb-4e28-bd8d-8ecdf927d937">

Reply:
if I add `[]`
<img width="652" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/91ad0bde-cb51-435c-8f8f-757d53e2b900">

tests are failed
<img width="897" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/b9f14db1-e571-4438-9bd7-19e84f2c7c24">